tests_new/integration_tests/core/environment/global_conftest.py

```diff
@@ -43,19 +44,20 @@ def session_env1(client1, group1, org1, session_id, testdata):
     envdata = testdata.envs['session_env1']
     env = None
     try:
-        env = create_env(client1, group1, org1['organizationUri'], envdata.accountId, envdata.region, tags=[session_id])
+        env = create_env(client1, group1, org1.organizationUri, envdata.accountId, envdata.region, tags=[session_id])
         yield env
     finally:
         if env:
             delete_env(client1, env)
 
 
 @pytest.fixture(scope='session')
-def session_env2(client1, group1, org1, session_id, testdata):
+def session_env2(client1, group1, group2, org2, session_id, testdata):
     envdata = testdata.envs['session_env2']
     env = None
     try:
-        env = create_env(client1, group1, org1['organizationUri'], envdata.accountId, envdata.region, tags=[session_id])
+        env = create_env(client1, group1, org2.organizationUri, envdata.accountId, envdata.region, tags=[session_id])
+        invite_group_on_env(client1, env.environmentUri, group2, ['CREATE_DATASET'])
```

Comment:
nit: should we import `ENVIRONMENT_INVITED` from `backend/dataall/core/permissions/services/environment_permissions.py` instead of static `CREATE_DATASET`?

Reply:
after looking at tests I see the intention of limiting permissions for invited group for the unauthorized tests - this I think should be fine